src/uu/pinky/src/platform/openbsd.rs

```diff
@@ -0,0 +1,17 @@
+// This file is part of the uutils coreutils package.
+//
+// For the full copyright and license information, please view the LICENSE
+// file that was distributed with this source code.
+
+// Specific implementation for OpenBSD: tool unsupported (utmpx not supported)
+
+use crate::uu_app;
+
+use uucore::error::UResult;
+
+pub fn uumain(args: impl uucore::Args) -> UResult<()> {
+    let _matches = uu_app().try_get_matches_from(args)?;
+
+    println!("unsupported");
```

Comment:
```suggestion
    println!("unsupported command on OpenBSD");
```

Reply:
and for other occurrences

Reply:
OK, fix message ("unsupported command on OpenBSD") for `pinky`, `uptime`, `users` and `who`.